### ISSUE TITLE
Support "pushtx"

### ISSUE THREAD

**Issue by priestc** [iquidus/explorer#77]
This explorer is pretty good but there is one feature that it is missing, and tat is "push_tx". This should be a pretty easy thing to implement. Just set up an API endpoint that takes a raw transaction and then pass it onto the node via "sendrawtransaction", then the txid should be returned to the user.

**Comment by TheHolyRoger** [comment]
I agree, broadcast TX page would be very useful.

**Comment by Tech1k** [comment]
I say that something similar to the Insight API and broadcast transaction page, is really needed for Iquidus.

**Comment by DigitalPandacoin** [comment]
Yes would love to see the API support broadcasting a transaction (pushtx)
It could be called with a POST and the content data is what will be forwarded (verbatim) to the wallet's sendrawtransaction RPC API
